### DISCUSSION TITLE
Option to reset habit progress

### DISCUSSION THREAD

**Discussion by siftnoorsingh** [iSoron/uhabits#1382] [Feature Requests]
I have habits that I dont want to delete or recreate again but it would be nice to have an option to 'Restart habit progress'. This should delete all the previous data added for that habit and start fresh

**Comment by LordJon** [comment]
This is seems to be a duplicate of https://github.com/iSoron/uhabits/issues/61
iSoron explains in this comment https://github.com/iSoron/uhabits/issues/61#issuecomment-199942384 why Loop habits doesn't has the feature.
